codegen/templates/vec.rs.tera

```diff
@@ -134,8 +134,19 @@ pub const fn {{ self_t | lower }}(
 /// This type uses 16 byte aligned SIMD vector type for storage.
 {%- else %}
 /// A {{ dim }}-dimensional vector.
-{%- endif %}
-#[derive(Clone, Copy)]
+{%- endif %}{% if not is_float %}
```

Comment:
It doesn't matter for this PR, more FYI,  you can add a hyphen inside `{%-` to trim preceding white-space which would allow you to have this`if` block on a separate line.